adapters/IronSource/IronSourceAdapter/ISMediationManager.m

```diff
@@ -139,6 +149,35 @@ - (void)presentInterstitialAdFromViewController:(nonnull UIViewController *)view
   [IronSource showISDemandOnlyInterstitial:viewController instanceId:instanceID];
 }
 
+- (void)loadBannerAdWithDelegate:
+            (nonnull id<GADMAdapterIronSourceBannerDelegate>)delegate
+                      viewController:(nonnull UIViewController *)viewController
+                      instanceID:(nonnull NSString *)instanceID
+                      bannerSize:(nonnull ISBannerSize*)size{
+    id<GADMAdapterIronSourceBannerDelegate> adapterDelegate = delegate;
+    if (adapterDelegate == nil) {
+      [GADMAdapterIronSourceUtils
+          onLog:[NSString
+                    stringWithFormat:@"requestBannerAdWithDelegate adapterDelegate is null."]];
+      return;
+    }
+    if ([self canLoadBannerInstance:instanceID]) {
+      [self setBannerDelegate:adapterDelegate toInstanceState:GADMAdapterIronSourceInstanceStateLocked];
+      [self addBannerDelegate:adapterDelegate forInstanceID:instanceID];
+      [GADMAdapterIronSourceUtils
+          onLog:[NSString
+                    stringWithFormat:@"ISMediationManager - loadBanner  for instance Id %@",
+                                     instanceID]];
+    
+      [IronSource loadISDemandOnlyBannerWithInstanceId:instanceID viewController:viewController size:size];
+    } else {
+      NSError *error = GADMAdapterIronSourceErrorWithCodeAndDescription(
+          GADMAdapterIronSourceErrorAdAlreadyLoaded,
+          @"Instance already loaded. Couldn't load another one in the same time!");
+        [adapterDelegate bannerDidFailToLoadWithError:error instanceId:instanceID];
+    }
```

Comment:
While this code is here, I noticed it is not for Android.
Can you confirm for each platform wether each of them support loading multiple banner ads using the same instance ID?

Reply:
So in the banner, we support the loading of the same instance ID.
So the second load will override the first load.
I removed the error from the banner.
Thanks